### ISSUE TITLE
Can't import pandas on Docker container

### ISSUE THREAD

**Issue by pddg** [pddg/galaxy_crawler#5]
Following exception was occured.

```
Traceback (most recent call last):
  File "/usr/local/bin/galaxy", line 11, in <module>
    load_entry_point('galaxy-crawler', 'console_scripts', 'galaxy')()
  File "/usr/local/lib/python3.8/site-packages/pkg_resources/__init__.py", line 489, in load_entry_point
    return get_distribution(dist).load_entry_point(group, name)
  File "/usr/local/lib/python3.8/site-packages/pkg_resources/__init__.py", line 2852, in load_entry_point
    return ep.load()
  File "/usr/local/lib/python3.8/site-packages/pkg_resources/__init__.py", line 2443, in load
    return self.resolve()
  File "/usr/local/lib/python3.8/site-packages/pkg_resources/__init__.py", line 2449, in resolve
    module = __import__(self.module_name, fromlist=['__name__'], level=0)
  File "/opt/galaxy/galaxy_crawler/cli.py", line 3, in <module>
    from galaxy_crawler.commands import root, crawl, db, load, clone
  File "/opt/galaxy/galaxy_crawler/commands/clone.py", line 6, in <module>
    import pandas as pd
  File "/usr/local/lib/python3.8/site-packages/pandas/__init__.py", line 55, in <module>
    from pandas.core.api import (
  File "/usr/local/lib/python3.8/site-packages/pandas/core/api.py", line 24, in <module>
    from pandas.core.groupby import Grouper, NamedAgg
  File "/usr/local/lib/python3.8/site-packages/pandas/core/groupby/__init__.py", line 1, in <module>
    from pandas.core.groupby.generic import (  # noqa: F401
  File "/usr/local/lib/python3.8/site-packages/pandas/core/groupby/generic.py", line 44, in <module>
    from pandas.core.frame import DataFrame
  File "/usr/local/lib/python3.8/site-packages/pandas/core/frame.py", line 115, in <module>
    from pandas.core.series import Series
  File "/usr/local/lib/python3.8/site-packages/pandas/core/series.py", line 4892, in <module>
    Series._add_series_or_dataframe_operations()
  File "/usr/local/lib/python3.8/site-packages/pandas/core/generic.py", line 10747, in _add_series_or_dataframe_operations
    from pandas.core import window as rwindow
  File "/usr/local/lib/python3.8/site-packages/pandas/core/window.py", line 13, in <module>
    import pandas._libs.window as libwindow
ImportError: Error loading shared library libstdc++.so.6: No such file or directory (needed by /usr/local/lib/python3.8/site-packages/pandas/_libs/window.cpython-38-x86_64-linux-gnu.so)
```

**Comment by pddg** [comment]
Use `python:3.7-slim-buster` instead of `python:3.7-alpine`

**Comment by pddg** [comment]
Fixed by c8c06a639ce979813a56d1cbc2f93e51f2e420e8
